deploy/openshift/clusterrole.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     app: kiali
     version: ${VERSION_LABEL}
 rules:
-- apiGroups: [""]
+- apiGroups: ["", "route.openshift.io"]
```

Comment:
split out the groups. We recently made changes to this yaml file to get all the apiGroups individually declared such that the "" group is separate from anything else.

Reply:
Ok, this will be updated.